rest-api/README.md

```diff
@@ -0,0 +1,120 @@
+# Sample code for fetching session data with Kobiton REST API
+## Prerequisites
+  - Node 
+    >Please visit here to install https://nodejs.org/en/download/.
```

Comment:
Inconsistent with others guideline. Please remove this line or add this to others.